### PR TITLE
Feature/fix zipkin span backport

### PR DESCRIPTION
**Pull request by samratdhillon**
Backport fix of CAMEL-16509

### CODE REVIEW

File: core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.support;
+
+/**
+ *
+ * @author     Samrat.Dhillon
```
Comment (davsclaus) [inline core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java:21]: Remove author tag

**Review by davsclaus — CHANGES_REQUESTED** [review]

File: core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java (hunk 1 above)
Comment (samratdhillon) [reply core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java]: @davsclaus pushed changes matching master. I see changes in master trigger a build. Is it possible to verify changes in maintenance branch.

**Review by davsclaus — APPROVED** [review]

File: core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java (hunk 1 above)
Comment (davsclaus) [reply core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java]: There CI just do a build and code format testing - no full test run. That is only for code merged.

Comment (davsclaus) [reply core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java]: And this is the danger of doing changes in core in a LTS release - eg we prefer not to. But this fixes a real bug for distributed tracing that is hard to "notice" due thread concurrency issues etc.

Comment (samratdhillon) [reply core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java]: @davsclaus Then please dont merge it yet. Let me rethink an easier way for LTS.

Comment (samratdhillon) [reply core/camel-support/src/main/java/org/apache/camel/support/CamelCopySafeProperty.java]: Closing this in lieu of PR#5435

**Pull request closed without merging**